http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20240117_create_app_chart_version_table.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="leonardo" xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+    <changeSet logicalFilePath="leonardo" author="ladieu" id="create_app_chart_version_table">
+        <createTable tableName="APP_CHART_VERSION">
+            <column name="chartName" type="VARCHAR(254)">
+                <constraints primaryKey="true"/>
+            </column>
+            <column name="chartVersion" type="VARCHAR(254)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="appVersion" type="VARCHAR(254)">
+                <constraints nullable="true"/>
+            </column>
+            <column name="timestamp" type="TIMESTAMP">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+        <addForeignKeyConstraint baseColumnNames="chartName" baseTableName="APP_CHART_VERSION"
+                                 constraintName="FK_APP_CHART_VERSION_APP_NAME" deferrable="false"
+                                 initiallyDeferred="false" onDelete="NO ACTION" onUpdate="NO ACTION"
+                                 referencedColumnNames="appName" referencedTableName="APP"/>
```

Comment:
Since the `chartName` column references `APP.appName`, would it be worth also calling this column "appName" instead of "chartName" to avoid having two terms for the same thing?

Reply:
Makes sense, fixed in 5eefc30f0a772cdd459746bb0218604049e15d2c

Reply:
Undid this (see https://github.com/DataBiosphere/leonardo/pull/4117#issuecomment-1900598800)